src/c/native_ledger_client_wrap.cc

```diff
@@ -0,0 +1,68 @@
+/* Copyright (c) 2018 Brave Software. Distributed under the MPL2 license.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+#include "src/c/native_ledger_client_wrap.h"
+
+namespace native_ledger {
+  Nan::Persistent<v8::FunctionTemplate> NativeLedgerClientWrap::constructor;
+
+  // Init
+  NAN_MODULE_INIT(NativeLedgerClientWrap::InitAll) {
+    v8::Local<v8::FunctionTemplate> ctor =
+      Nan::New<v8::FunctionTemplate>(NativeLedgerClientWrap::New);
+    constructor.Reset(ctor);
+    ctor->InstanceTemplate()->SetInternalFieldCount(0);
+    ctor->SetClassName(Nan::New("LedgerNode").ToLocalChecked());
```

Comment:
c++ I'd use `node`, but on the JS side I'd probably just refer to it as Ledger. Just a nit though